### ISSUE TITLE
Clean up old issues

### ISSUE THREAD

**Issue by endofcake** [msmhq/msm#356]
There are a bunch of issues all the way back to 2012 that no one will likely look at. I suggest most of them be closed.
@marcuswhybrow

**Comment by M-D-M** [comment]
A request I can agree with.  I propose anything older than 2015 issues be closed.  Minecraft dev has kinda slowed down since then, I think.

**Comment by morbeo** [comment]
What about PRs?

**Comment by endofcake** [comment]
Good point @morbeo . The short answer is I don't really know, would be good to discuss it though.

**Comment by morbeo** [comment]
Perhaps start a cleanup project here in the github interface where the issues&PR would be processed kanban style. Not sure what qualifies a reviewer tho.

**Comment by kjkazinski** [comment]
I would also like to see a new version released

**Comment by morbeo** [comment]
@kjkazinski A new version like next step in the semantic versioning or new major release? Major release entail **a lot** of things not all of which are reasonable or necessary currently. New minor updates would be released when pull requests start flowing.

The lack of core community, maintainer attention and disproportionate amount of people needing help vs those that want to contribute are holding msm back.

**Comment by kjkazinski** [comment]
I was thinking it would be nice to have even a minor version update.  My GIT skills are not the great so I am never sure if I have the current version

**Comment by Maxiride** [comment]
Is this repo abandoned? Even this issue is already a year old without further development.

**Comment by endofcake** [comment]
@Maxiride , time is scarce unfortunately. Are there any immediate problems that need solving?

**Comment by endofcake** [comment]
I'll go ahead and close the old issues though, unless they report severe problems.

**Comment by Maxiride** [comment]
@endofcake not at all =) I was just browsing for managing solutions. It's all fine as I eventually went for MineOs

Thanks for replying though!
